test/unit/data/exporter/index.test.js

```diff
@@ -173,10 +173,16 @@ describe('Exporter', function () {
             } = require('../../../../core/server/data/exporter/table-lists.js');
 
             const nonSchemaTables = ['migrations', 'migrations_lock'];
+            const requiredTables = schemaTables.concat(nonSchemaTables);
+            // NOTE: You should not add tables to this list unless they are temporary
+            const ignoredTables = ['temp_member_analytic_events'];
```

Comment:
We don't use the blocklist in the application code anywhere.

This only exists so that any addition to the schema will cause the test to fail still. Without it we would not be able to know that `temp_member_analytic_events` table should not be in the `BACKUP_TABLES` array

Reply:
@ErisDS Heads up - I could find a way to refactor the tests without needing this.

The entire point of this test is to stop people from accidentally adding a table, and forgetting to add it to the `BACKUP_TABLES` or `TABLES_ALLOWLIST` - so to do that we need to compare the schema with those lists and error if there is a disrepancy.

I couldn't see another way around this, without knowing to ignore this table!